client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -143,15 +163,32 @@ class PodcastDetails extends Component {
 					>
 						<h1>{ translate( 'Podcasting Settings' ) }</h1>
 					</HeaderCake>
-					<Card>
-						{ this.renderTextField( {
-							key: 'podcasting_title',
-							label: translate( 'Title' ),
-						} ) }
-						{ this.renderTextField( {
-							key: 'podcasting_subtitle',
-							label: translate( 'Subtitle' ),
-						} ) }
+					<Card className="podcasting-details__wrapper">
+						<FormFieldset className="podcasting-details__category-selector">
+							<QueryTerms siteId={ siteId } taxonomy="category" />
+							{ /* Add name to TermTreeSelector, label, and explanation */ }
+							<FormLabel htmlFor="">{ translate( 'Podcast Category' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Podcast category 3' )` *ES Score: 20*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/podcasting-category-selector)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).